src/colvarscript.h

```diff
@@ -296,13 +437,83 @@ extern "C" {
            0, 1,
            { },
            if (objc < 3) {
-            script->set_str_result(cvm::proxy->units);
+            script->set_result_str(cvm::proxy->units);
             return COLVARS_OK;
            } else {
             return cvm::proxy->set_unit_system(script->obj_to_str(objv[2]) , false);
            }
            )
 
+  CVSCRIPT(cv_printframelabels,
+           "Print the labels that would be written to colvars.traj",
+           0, 0,
+           { },
+            std::ostringstream os;
+            script->colvars->write_traj_label(os);
+            script->set_result_str(os.str());
+            return COLVARS_OK;
+           )
+
+  CVSCRIPT(cv_printframe,
+           "Print the values that would be written to colvars.traj",
+           0, 0,
+           { },
+            std::ostringstream os;
+            script->colvars->write_traj(os);
+            script->set_result_str(os.str());
+            return COLVARS_OK;
+           )
+
+  CVSCRIPT(cv_frame,
+           "Get or set current frame number",
+           0, 1,
+           { },
+            if (objc == 2) {
+              long int f;
+              int error = script->proxy->get_frame(f);
+              if (error == COLVARS_OK) {
+                script->set_result_str(cvm::to_str(f));
+                return COLVARS_OK;
+              } else {
+                script->set_error_msg("Frame number is not available");
+                return COLVARSCRIPT_ERROR;
+              }
+            } else if (objc == 3) {
+              // Failure of this function does not trigger an error, but
+              // returns nonzero, to let scripts detect available frames
+              int error = script->proxy->set_frame(strtol(script->obj_to_str(objv[2]), NULL, 10));
+              script->set_result_str(cvm::to_str(error == COLVARS_OK ? 0 : -1));
+              return COLVARS_OK;
+            }
+           )
+
+  CVSCRIPT(cv_colvar,
+           "Access colvar-specific commands: ",
+           2, 4,
+           {"name (str) - colvar name"},
```

Comment:
This string should also contain documentation for the optional parameters, which could also exceed 4, e.g.:
`cv colvar xi set feature on|off`

Notice how the commands implemented so far in the `colvarscript-files` branch are simpler (one argument at most).